omnisafe/algorithms/model_based/base/ensemble.py

```diff
@@ -179,39 +249,54 @@ def __init__(
         cost_size: int,
         ensemble_size: int,
         predict_reward: bool,
-        predict_cost: bool | None = None,
+        predict_cost: bool = False,
         hidden_size: int = 200,
         learning_rate: float = 1e-3,
         use_decay: bool = False,
     ) -> None:
         """Initialize network weight."""
         super().__init__()
 
-        self._state_size = state_size
-        self._reward_size = reward_size
-        self._cost_size = cost_size
-        self._predict_reward = predict_reward
-        self._predict_cost = predict_cost
+        self._state_size: int = state_size
+        self._reward_size: int = reward_size
+        self._cost_siz: int = cost_size
```

Comment:
```suggestion
        self._cost_size: int = cost_size
```

Reply:
Won't this cause exceptions in CI?

Reply:
Well, I don't know either but I fixed it.